otx/algorithms/detection/adapters/mmdet/datasets/tiling.py

```diff
@@ -356,7 +356,7 @@ def __getitem__(self, idx):
         result = copy.deepcopy(self.tiles[idx])
         dataset_idx = result["dataset_idx"]
         x_1, y_1, x_2, y_2 = result["tile_box"]
-        ori_img = self.cached_results[dataset_idx]["dataset_item"].media.numpy
+        ori_img = self.cached_results[dataset_idx]["img"]
```

Comment:
What's the difference and impact of this change?

Reply:
I've checked this through unit test and running tiling through cli. Both are same